src/Momento.Sdk/Internal/ScsDataClient.cs

```diff
@@ -78,7 +78,7 @@ public ScsDataClient(IConfiguration config, string authToken, string endpoint, T
 
     public async Task<CacheSetResponse> SetAsync(string cacheName, byte[] key, byte[] value, TimeSpan? ttl = null)
     {
-        return await this.SendSetAsync(cacheName, value: value.ToByteString(), key: key.ToByteString(), ttl: ttl);
+        return await this.SendSetAsync(cacheName, key: key.ToByteString(), value: value.ToByteString(),  ttl: ttl);
```

Comment:
thanks for changing this!  it's definitely more consistent and readable this way.

Reply:
Not sure how this was not caught in the test cases. The test cases use Utils.NewGuidByteArray() to generate keys and values, so ideally, it should have been captured there!

Reply:
for posterity: we discussed this on a zoom - it's because of the keyword args here.  even though they were passed in a weird order, it worked, because of the keyword args.